src/pages/ItemList.js

```diff
@@ -15,11 +15,14 @@ const ONE_MINUTE = ONE_SECOND * 60;
 const ONE_HOUR = ONE_MINUTE * 60;
 const ONE_DAY = ONE_HOUR * 24;
 
-
 function ItemList({ token }) {
   const [docs, setDocs] = useState([]);
+  const [filteredList, setFilteredList] = useState([]);
+  const [searchInput, setSearchInput] = useState('');
   const [now, setNow] = useState(Date.now());
   const [isLoading, setIsLoading] = useState(false);
+  let list = [];
```

Comment:
Within "functional programming," which is the style that a lot of React is built around, we usually try to avoid mutation because it can lead to non-deterministic behavior (i.e. you can do the same thing but get different results). Reassignment like we're doing with `list` is a form of mutation.

If we think about it, 

```javascript
let list = [];
filteredList ? (list = filteredList) : (list = docs);
```
is a logical equivalent to

`const list = filteredList ? filteredList : docs;`

which is a logical equivalent to

`const list = filteredList || docs;`

which is a much shorter phrase that doesn't mutate anything :-)

Using logical operators like as above may be a little confusing, so let me know if you're not entirely grokking their logical equivalence.

If you'd like to read more about functional programming, here's a nice little primer - https://www.freecodecamp.org/news/an-introduction-to-the-basic-principles-of-functional-programming-a2c2a15c84/